telemetry/definitions/commonDefinitions.json

```diff
@@ -2609,50 +2589,74 @@
             ],
             "passive": true
         },
+
         {
-            "name": "awsq_serviceInvocation",
-            "description": "Client side invocation of the weaverbird service for solving software engineering tasks",
-            "metadata": [
-                {
-                    "type": "awsqAssignClickCount"
-                },
-                {
-                    "type": "awsqGenerateCodeIterationCount"
-                },
-                {
-                    "type": "awsqGenerateCodeClickCount"
-                },
-                {
-                    "type": "awsqApproachIterationCount"
-                },
-                {
-                    "type": "awsqIsRegeneratedCode"
-                },
-                {
-                    "type": "awsqFilesReviewedCount"
-                },
-                {
-                    "type": "awsqIsAcceptedCodeChanges"
-                },
-                {
-                    "type": "awsqRepositorySize"
-                },
-                {
-                    "type": "awsqFilesChangedCount"
-                },
-                {
-                    "type": "awsqThumbsUpCount"
-                },
-                {
-                    "type": "awsqThumbsDownCount"
-                },
-                {
-                    "type": "awsqIsApproachAccepted"
-                },
-                {
-                    "type": "awsqServiceErrorsCount"
-                }
-            ]
+            "name": "awsq_assignCommand",
+            "description": "User have triggered /assign",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqAssignCommandCount" }]
+        },
+        {
+            "name": "awsq_codeReGeneration",
+            "description": "User invoked the weaverbird agent to re-engage in Code gen",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqCodeReGenerationCount" }]
+        },
+        {
+            "name": "awsq_codeGenerateClick",
+            "description": "User has clicked on Generate Code after have iterated through the Approach phase",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqGenerateCodeClickCount" }]
+        },
+        {
+            "name": "awsq_approach",
+            "description": "User received an Approach",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqApproachGeneratedCount" }]
+        },
+        {
+            "name": "awsq_isRegeneratedCode",
+            "description": "User has re-generated code as a part of code generation portion",
+            "metadata": [{ "type": "enabled" }]
+        },
+        {
+            "name": "awsq_filesReviewed",
+            "description": "User reviewed X files after Code Generation phase",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqFilesReviewedCount" }] 
```

Comment:
`awsqFilesReviewedCount` is unnecessary because the metric is defined with unit:Count.

Reply:
removed